about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Microverse</title>
+    <link rel="stylesheet" href="./styles/about/about.css">
+    <link rel="shortcut icon" href="./images/microverse.png" type="image/x-icon">
+</head>
+<body>
+    <main class="container">
+        <header class="container__header">
+            <nav class="hamburger-menu">
```

Comment:
- [ ] I couldn't be able to open the menu with the burger Icon, please fix this. 🐶